samples/tab-request-approval/nodejs/views/UserNotification.ejs

```diff
@@ -34,7 +34,7 @@
             <div id="createtask" class="tab-pane fade in active">
                 <div class="mb">
                     <div class="form-group fg">
-                        <mgt-teams-msal2-provider client-id="9811a0fb-ea05-46c7-b631-9c24f55ffe0c"
+                        <mgt-teams-msal2-provider client-id="df99fae2-35ed-46a7-b946-c1b42ae885fb"
```

Comment:
remove client id. It should come from server only

Reply:
fixed